Makefile

```
@@ -40,18 +40,32 @@ install_script_agent7.sh: install_script.sh.template
 	chmod +x $@
 
 pre_release_%:
+	$(eval CUR_VERSION=$(shell awk -F "=" '/^install_script_version=/{print $$NF}' install_script.sh.template))
+	$(eval CUR_VERSION=$(shell echo "${CUR_VERSION}" | sed -e 's|.post||'))
```

We can compact it into a one liner.
```suggestion
	$(eval CUR_VERSION=$(shell awk -F "=" '/^install_script_version=/{print $$NF}' install_script.sh.template | sed -e 's|.post||'))
```